.github/workflows/deploy.yml

```diff
@@ -30,18 +30,14 @@ jobs:
     if: github.ref == 'refs/heads/production'
 
     steps:
-      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+      - uses: actions/checkout@v3
         with:
-          fetch-depth: 0
           token: ${{ secrets.OS_BOTIFY_TOKEN }}
 
       - uses: Expensify/App/.github/actions/composite/setupGitForOSBotify@main
         with:
           GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
 
-      - name: Checkout production branch
```

Comment:
This wasn't needed because the job only runs if `github.ref === 'refs/heads/production'`, and actions/checkout will checkout `github.ref` by default.

Reply:
I added `ref: production` above just to be more explicit here